packages/gestalt/src/Table.css

```diff
@@ -14,6 +14,12 @@
   composes: paddingX3 from "./boxWhitespace.css";
 }
 
-.thead {
-  composes: borderBottom from "./Borders.css";
+.sticky tr th {
+  background-color: #fff;
+  position: sticky;
+  top: 0;
+}
+
+.thead tr:last-child th {
+  border-bottom: 1px solid #efefef;
```

Comment:
@PayalModi borders should now be `#ddd` - see #776 /cc @ashleyseto

We also had this with the previous `composes: borderBottom from "./Borders.css";`

Reply:
I get this error when trying to use compose for this class: `Error: composition is only allowed when selector is single :local class name not in ":local(.thead) tr:last-child th"`. But I updated to use #ddd for this border - thanks for catching that!